File: code-coaches/easy/kaleidoscopes/Kaleidoscopes.cs
using System;

public class Kaleidoscopes
{
	static double GetTotal(int numKaleidoscopes)
	{
	    double price = 5.0;
        double tax = 1.07;
        double subtotal = price;

        if (numKaleidoscopes > 1)
        {
            double discount = 0.9;
            subtotal = (double) numKaleidoscopes * price * discount;
        }

        double total = subtotal * tax;

        return Math.Round(total, 2, MidpointRounding.AwayFromZero);
	}

	static void Main(string[] args) {
		int numKaleidoscopes = int.Parse(Console.ReadLine());
		Console.WriteLine(GetTotal(numKaleidoscopes));
	}

}

File: code-coaches/easy/kaleidoscopes/kaleidoscopes.rb
def get_total(num_kaleidoscopes)
  price = 5
  tax = 1.07

  if num_kaleidoscopes > 1
    discount = 0.9
    subtotal = num_kaleidoscopes * price * discount
  else
    subtotal = price
  end

  total = subtotal * tax
  total.round(2)
end

def main
  num_kaleidoscopes = gets.chomp.to_i
  puts get_total(num_kaleidoscopes)
end

if __FILE__ == $0
  main()
end
